osmomath/export_test.go

```diff
@@ -1,6 +1,14 @@
 package osmomath
 
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
 var (
 	EulersNumber = eulersNumber
 	TwoBigDec    = twoBigDec
 )
+
+// 2^128 - 1, needs to be the same as gammtypes.MaxSpotPrice
+// but we can't directly import that due to import cycles.
+// Hence we use the same var name, in hopes that if any change there happens,
+// this is caught via a CTRL+F
+var MaxSpotPrice = sdk.NewDec(2).Power(128).Sub(sdk.OneDec())
```

Comment:
Would having separate constants or utils package help mitigate the import cycles?

Reply:
It would have to be in its own independent go.mod, which seems pretty annoying for now